keygen/src/keygen.rs

```diff
@@ -14,19 +15,78 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                 .takes_value(true)
                 .help("Path to generated file"),
         )
+        .subcommand(
+            SubCommand::with_name("new")
+                .about("Generate new keypair file")
+                .arg(
+                    Arg::with_name("outfile")
+                        .short("o")
+                        .long("outfile")
+                        .value_name("PATH")
+                        .takes_value(true)
+                        .help("Path to generated file"),
+                ),
+        )
+        .subcommand(
+            SubCommand::with_name("print")
```

Comment:
This is a confusing interface. I'd expect `solana-keygen print` to a print a `pkcs8` string for the keypair.  And maybe something like `solana-keygen print --infile=my_keypair.txt --public-key` to print the base58 pubkey string.  I don't see the point of an `outfile` parameter at this time.

Reply:
I un-resolved the conversation with @mvines, if you'd like to refer to his suggestions.
I rather agree with you, except about the pubkey outfile. One of the main motivators of this utility in the first place was being able to easily move pubkeys between devices without providing keypair access.

Perhaps it's simply changing the "print" subcommand to "pubkey"?